src/utils/fetchRequest.ts

```diff
@@ -0,0 +1,134 @@
+import { getVersionInfo } from './versionUtils.js';
+
+export interface FetchPolicy {
+  handle(
```

Comment:
Maybe we should add a name for the policy so we can reference these policies by name to add or remove them?